layers/+lang/ruby/README.org

```diff
@@ -133,6 +133,7 @@ When =ruby-test-runner= equals =rspec=.
 | Key binding | Description                        |
 |-------------+------------------------------------|
 | ~SPC m t a~ | run all specs                      |
+| ~SPC m t b~ | run current spec file              |
 | ~SPC m t c~ | =rspec-verify-continue=            |
```

Comment:
As you're working on this layer, it may be nice to also use sentences to describe other bindings in this array :-) It's just an idea, your current PR is fine as-is.